misc/git-hooks/commit-msg

```diff
@@ -10,44 +10,21 @@ function rm_tmp_file {
 
 trap rm_tmp_file EXIT
 
-while : ; do
-  # Sanitize file first, by removing leading lines that are empty or start with a hash,
-  # as `convco` currently does not do it automatically (but git will)
-  echo -n "" > "$tmp_file"
-  body_detected=""
-  while read -r line ; do
-    # skip any initial comments (possibly from previous run)
-    if [ -z "${body_detected:-}" ] && { [[ "$line" =~ ^#.*$ ]] || [ "$line" == "" ]; }; then
-      continue
-    fi
-    body_detected="true"
-
-    echo "$line" >> "$tmp_file"
-  done < "$1"
-
-  # We have a sanitized version in "$tmp_file" now, move it the original
-  cp "$tmp_file" "$1"
-
-
-  # Run convco, start preparing a buffer we will display to the user,
-  # if lint fails.
-  echo -n "# " > "$tmp_file"
-  if convco check < "$1" 2>> "$tmp_file" ; then
-     break
+# Sanitize file first, by removing leading lines that are empty or start with a hash,
+# as `convco` currently does not do it automatically (but git will)
+echo -n "" > "$tmp_file"
+body_detected=""
+while read -r line ; do
+  # skip any initial comments (possibly from previous run)
+  if [ -z "${body_detected:-}" ] && { [[ "$line" =~ ^#.*$ ]] || [ "$line" == "" ]; }; then
+    continue
   fi
+  body_detected="true"
```

Comment:
Where is `body_detected` used later on? I can't find it. Is there some implicit transfer to the caller?

Reply:
Leftover that I overlooked.